laravel-app/public/assets/js/followUnfollow.js

```diff
@@ -0,0 +1,51 @@
+$(document).ready(function () {
+    $('.follow-button').each(function () {
+        const button = $(this);
+        const isFollowing = button.data('isFollowing') === true;
+        updateButtonText(button, isFollowing);
+    });
+
+    $(document).on('click', '.follow-button', function () {
+        const button = $(this);
+        const userId = button.data('user-id');
+        const isFollowing = button.data('isFollowing') === true;
```

Comment:
why --- true? what is the data in the button? is it true and false?

Reply:
yes sir, the value is bool assigned in the blade

Reply:
so remove the === true. it will still work

Reply:
That is correct sir, I apologize for this oversight. Thank you ! ❤️

Reply:
Done sir, fixed assignment logic for boolean value. Thank you